cmd/build/v2/config.go

```diff
@@ -49,12 +58,26 @@ func getConfig(registry configRegistryInterface, gitRepo configRepositoryInterfa
 	if err != nil {
 		oktetoLog.Infof("error trying to get directory: %w", err)
 	}
+
+	enableSmartBuilds := true
+	enableSmartBuildsStr := os.Getenv(OktetoEnableSmartBuilds)
+	if enableSmartBuildsStr != "" {
+		smartBuildEnabledBool, err := strconv.ParseBool(enableSmartBuildsStr)
+		if err != nil {
+			oktetoLog.Infof("error trying to get feature flag %s: %w", OktetoEnableSmartBuilds, err)
```

Comment:
Agree with changing it into warning. Also, Should we encapsulate this code into a function and make the test function to focus on just that? I think the test is getting bigger and bigger and it can be hard to read if we continue adding use cases without having an specific tests

Reply:
27fbc6b3fe2a8d8a1d6208e1274c004e4bf1af35

Reply:
@jLopezbarb 391fa8f2661001c76650f84bfaa8634fa84497f1 to move feature flag retrieval